api/ProjectRouter.js

```diff
@@ -0,0 +1,127 @@
+const router = require('express').Router();
+
+const Projects = require('../data/helpers/projectModel');
+const Actions = require('../data/helpers/actionModel.js');
+
+router.get('/', (req, res) => {
+    Projects.get()
+    .then( projects => {
+        res.status(200).json(projects)
+    })
+    .catch(error => {
+        res.status(500).json({ error: "There was an error trying to retrieve the projects from the database." })
+    })
+})
+
+router.post('/', (req, res) => {
+    const { name, description } = req.body;
+    if(!name || !description){
+        res.status(400).json({ message: "Project name and description is required." })
+    } else {
+        
+    Projects.insert({ name, description })
+    .then( newProject => {
+        res.status(201).json(newProject)
+    })
+    .catch(error => {
+        res.status(500).json({ error: "There was an error trying to post a new project to the database." })
+    })
+    }
+})
+
+router.put('/:id', (req, res) => {
+    const id = req.params.id;
+    const changes = req.body;
+    const { name, description } = req.body;
+
+    if(!name || !description){
```

Comment:
This isn't totally _necessary_, it's up to you how it functions, just keep in mind the frontend needs to match up too